apps/extension/client/package.json

```diff
@@ -81,6 +81,16 @@
           "type": "boolean",
           "default": true,
           "description": "Enable nesting the generated typegen files."
+        },
+        "xstate.theme": {
+          "type": "string",
+          "enum": [
+            "auto",
+            "dark",
+            "light"
+          ],
+          "default": "auto",
+          "description": "Color theme for the visual editor. Auto syncs the vscode theme with the editor theme."
```

Comment:
```suggestion
          "description": "Color theme for the visual editor. Auto syncs the VS Code theme with the editor theme."
```

Reply:
I think this ⬆️ would be a good change, other than that it LGTM 😊